ddl/backfilling_scheduler.go

```diff
@@ -288,6 +299,12 @@ func (b *ingestBackfillScheduler) setupWorkers() error {
 		return errors.Trace(errors.New("cannot get lightning backend"))
 	}
 	b.backendCtx = bc
+	mgr, err := ingest.NewCheckpointManager(b.ctx, bc, b.sessPool, job.ID,
```

Comment:
It shouldn't set the manager in distribute case.

Reply:
Updated.